python/feathub/common/config.py

```diff
@@ -89,22 +89,41 @@ def __init__(
         self.validator = validator
 
 
+COMMON_PREFIX = "common."
+
+TIMEZONE_CONFIG = COMMON_PREFIX + "timeZone"
+TIMEZONE_DOC = (
+    "The Region ID that represents a timezone to be used during "
+    "timestamp-related data type conversion operations. Region IDs "
+    "must have the form 'area/city', such as 'America/Los_Angeles'."
+)
+
+# TODO: Add document describing the configuration options available
+#  in Feathub.
+common_config_defs: List[ConfigDef] = [
+    ConfigDef(
+        name=TIMEZONE_CONFIG,
+        value_type=str,
+        description=TIMEZONE_DOC,
+        default_value=str(tzlocal.get_localzone()),
```

Comment:
The returned timezone of `datetime.datetime.now` is "CST" rather than "Asia/Shanghai" in my case, and timezones in formats like "CST" is not supported in Flink `table.local-time-zone` configuration yet.